source/_integrations/modbus.markdown

```diff
@@ -1013,6 +1013,10 @@ sensors:
       description: The maximum allowed value of a sensor. If value > max_value --> max_value. Can be float or integer
       required: false
       type: float
+    nan_value:
+      description: If a modbus sensor has a vendor defined NaN value, this value can be set here as hex string (e.g. `0xFFFFFFFF`) and, if triggered, the sensor becomes `unavailable`.
```

Comment:
In the code PR it says that the Na value can be a int, float or he. value. This should be matched here.

Reply:
the PR is string input also, first version with int/float/hex had issues. Should be consistent with documentation here

Reply:
The e.g. does not correspond to the code, you demand a specific format, so please inform the user clearly.

Reply:
you're right, thanks!